stripper/ze_FFVII_Mako_Reactor_v5_3_v5.cfg

```diff
@@ -3985,4 +3985,18 @@ modify:
 		"OnNewGame" "godgaymer_fuckerAddOutputmins -134.5 -134.5 -8051-1"
 		"OnNewGame" "godgaymer_fuckerAddOutputmaxs 134.5 134.5 8051-1"
 	}
-}
\ No newline at end of file
+}
+
+;attempt to fix zombies infecting humans on the train?
+modify:
+{
+	match:
+	{
+		"classname" "trigger_once"
+		"hammerid" "824509"
+	}
+	delete:
+	{
+		"OnStartTouch" "consolaCommandmp_freezetime 00-1"
```

Comment:
A better fix for the mako start (at least in my opinion) would be for either the infection timer to just be extended for 1 extra second (only vauff/snowy could do this, since it is a protected cvar, and not one the map can set), so zombies spawn in right as the tp activates to bring them away, or have the tp just activate a second earlier. This way it keeps the the server freezetime setting constant across maps while still fixing the issue.

Reply:
This doesn't work at all lol, infection timer is tied to the start of the round (after freezetime ends), so the freezetime value is irrelevant here.

Like Ice said, we could just tweak the infection timer server side if we wanted to patch this out. However I've always seen it as a fun quirk of the map, especially since mako originally used to work with classic spawn to infect an entire train. Gives a small taste of that if you're lucky enough to manage to infect someone (or unlucky enough to be on the receiving end).

Reply:
This doesn't work at all lol, infection timer is tied to the start of the round (after freezetime ends), so the freezetime value is irrelevant here.

Like Ice said, we could just tweak the infection timer server side if we wanted to patch this out. However I've always seen it as a fun quirk of the map, especially since mako originally used to work with classic spawn to infect an entire train. Gives a small taste of that if you're lucky enough to manage to infect someone (or unlucky enough to be on the receiving end).